packages/mock-server/src/schema/Transaction.js

```diff
@@ -71,6 +71,15 @@ const TransactionQueryResolvers = {
 };
 
 const TransactionMutationResolvers = {
+  deleteTransactions: (_, { transactions }) => {
+    transactions.forEach(({ id: deleteId }) => {
+      const idx = TransactionData.findIndex(({ id }) => deleteId === id);
+      TransactionData.splice(idx, 1);
+    });
+
+    return transactions;
+  },
+
```

Comment:
I should've done this is a different pr :( - deleting transaction mock and handler in the mock server